src/app/listingpage/productcard/productcard.component.html

```diff
@@ -0,0 +1,58 @@
+ <div class="card-deck">
+  <div class="card">
+    <img class="card-img-top" src="https://cdn.pixabay.com/photo/2016/11/22/23/38/apartment-1851201_1280.jpg" alt="Card image cap">
+    <div class="card-body row">
+      <div class="col-sm-7" style="text-align: left;">
+        <h6 class="card-title">Premium set sofa with chair</h6>
```

Comment:
you can use h3 onwards throughout your page - why are you starting from h6 directly?

Reply:
I would say, it was a need of the look and feel of page.

<img width="960" alt="look" src="https://user-images.githubusercontent.com/113106308/210559051-b4c45ab9-0544-4a46-995f-154c24b5f46e.png">